values.yaml

```diff
@@ -40,6 +40,17 @@ podAnnotations: {}
 podSecurityContext: {}
   # fsGroup: 2000
 
+podDisruptionBudget:
```

Comment:
Instead of having multiple top level pod* configuration options, I've been working towards nesting all of these under the `pod` key. Would you mind making those changes to keep it consistent with the future direction?

Additionally, for changes that aren't directly related to the configuration of the relay proxy, I have been trying to keep their configuration structure as flexible as possible. What do you think about making the `pod.disruptionBudget` value default to an empty object. 

That would mean your enabled check can instead check if there are values defined. If so, you can then define the basic structure of the yaml file and directly render their configuration. Thoughts?

Reply:
Hey @keelerm84 ,

Was just working on these changes. I am happy to move it under top-level pod, however, for the configuration part I think it might be best to keep it as is. One of the reasons is the use of: `{{- include "ld-relay.selectorLabels" . | nindent 6 }}` . If we were to just render the whole yaml from the values object then this function couldn't be used and the user would have to explicitly know what the selector labels would be. Also, there are only 2 options for configuration in the pdb so I don't think it would matter too much on allowing for flexibility here. I have looked at a few other charts and they all seem to do it this way. Happy to hear your thoughts :)

Reply:
That seems fair to me. Thank you for having explored that and raising the concern!

Reply:
Awesome, thanks @keelerm84 . All changes have been made